docs/scripts/generate_plugin_docs.py

```
@@ -993,29 +1020,96 @@ def generate_all_docs(self, readme_content: str):
+            logger.info(
+                f"Generated {len(self._plugin_model_cards)} plugin model cards"
+            )
 
         logger.info("Plugin documentation generated successfully!")
 
 
+class LabsDocGenerator(PluginDocGenerator):
+    """Generates Labs documentation from the fiftyone-labs repository."""
+
+    _ecosystem_subdir = "labs_ecosystem"
+    _SKIP_NAMES = {"labs_panel"}
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)